code/modules/power/cable.dm

```diff
@@ -28,7 +28,7 @@ By design, d1 is the smallest direction and d2 is the highest
 	on_blueprints = TRUE
 	var/datum/powernet/powernet
 	name = "power cable"
-	desc = "A flexible, superconducting cable for heavy-duty power transfer."
+	desc = "A flexible, superconducting insulated cable for heavy-duty power transfer."

```

Comment:
The desc change was half-meme for the people who were claiming that Nanotrasen just paints live cables.